src/plugin.ts

```diff
@@ -2,7 +2,7 @@
 // Distributed under the terms of the Modified BSD License.
 
 import {
-  JupyterLab, JupyterLabPlugin
+  JupyterLab, JupyterFrontEndPlugin
```

Comment:
Shouldn't it be `JupyterFrontEnd` instead of `JupyterLab` for 1.0?

Reply:
Hence this change, I am not sure I understand your comment?

Reply:
Oh sorry, you are talking about the first element imported. Thanks for the review, I will change it. Weird that it did build locally.

Reply:
Mmmh, it does not compile if I change `JupyterLab` with `JupyterFrontEnd`:
```
src/plugin.ts:57:11 - error TS7053: Element implicitly has an 'any' type because expression of type '"_rightHandler"' can't be used to index type 'IShell'.
  Property '_rightHandler' does not exist on type 'IShell'.

57           app.shell['_rightHandler'].sideBar.tabCloseRequested.connect((sender : any, tab : any) => {
             ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/plugin.ts:69:23 - error TS2339: Property 'expandRight' does not exist on type 'IShell'.

69             app.shell.expandRight();
                         ~~~~~~~~~~~
```